comb_spec_searcher/comb_spec_searcher.py

```diff
@@ -662,13 +636,13 @@ def _expand_classes_for(
         Will expand classes for `expansion_time` seconds.
 
         It will return a pair (bool, time), where the bool is True if there are
-        more classes to expand, False otherwise. The `time` is the initial time
-        for checking whether to post a status update.
+        more classes to expand, False otherwise. The `time` is the time of the last
```

Comment:
time of -> time since

Reply:
We actually return `status_start = time.time()`. The variable is updated just after the status update is logged so it indicate the time when the last status update was performed not the amount of time that since the last status update

Reply:
I see, still a small grammar issue, I'll fix it.